include/swift/SILOptimizer/Utils/PartitionUtils.h

```diff
@@ -196,7 +211,7 @@ class Partition {
       return;
     canonical = true;
 
-    std::map<signed, unsigned> relabel;
+    std::map<unsigned, unsigned> relabel;
```

Comment:
oh no now I can't tell what this means at all. can your next PR at least include those typealiases for `unsigned` and `signed` to give them a better name?

Reply:
ok, performing this refactor!